lib/runner.js

```diff
@@ -928,7 +951,7 @@ Runner.prototype.run = function(fn) {
     if (rootSuite.hasOnly()) {
       rootSuite.filterOnly();
     }
-    self.started = true;
+    self.state = constants.STATE_RUNNING;
```

Comment:
Is setting runner's state not too early, in case of `delay`?

Reply:
not sure.  will see.